index.html

```diff
@@ -1 +1 @@
-<h1>hey </h1>
\ No newline at end of file
+<h1>This is a main perfect feature </h1>
```

Comment:
This should be all capitalized
```suggestion
<h1>THIS IS A MAIN PERFECT FEATURE</h1>
```

Reply:
No it was perfect